tests/framework/utils/k8s_helper.go

```diff
@@ -30,13 +32,18 @@ import (
 	"time"
 
 	"github.com/coreos/pkg/capnslog"
+	"github.com/pkg/errors"
 	rookclient "github.com/rook/rook/pkg/client/clientset/versioned"
 	"github.com/rook/rook/pkg/clusterd"
 	"github.com/rook/rook/pkg/util/exec"
 	"github.com/stretchr/testify/require"
 	apps "k8s.io/api/apps/v1"
 	v1 "k8s.io/api/core/v1"
+<<<<<<< HEAD
 	"k8s.io/apimachinery/pkg/api/errors"
+=======
+	kerrors "k8s.io/apimachinery/pkg/api/errors"
+>>>>>>> c15499ed7... ceph: use the correct snapshot controller version in the tests
```

Comment:
Have merge conflicts here!

Reply:
We should make sure these didn't make it into master also

Reply:
Fixed now, thanks